erdpy/accounts.py

```diff
@@ -76,11 +82,55 @@ def get_secret_key(self) -> bytes:
             raise LedgerError("cannot get seed from a Ledger account")
         return unhexlify(self.secret_key)
 
+    def sign_transaction(self, transaction: ITransaction) -> str:
+        secret_key = self.get_secret_key()
+        signing_key: Any = nacl.signing.SigningKey(secret_key)
+
+        data_json = transaction.serialize()
+        signed = signing_key.sign(data_json)
+        signature = signed.signature
+        signature_hex = signature.hex()
+        assert isinstance(signature_hex, str)
+
+        return signature_hex
```

Comment:
Consider the following instead:
```suggestion
        data_json = transaction.serialize()
        signed = signing_key.sign(data_json)
        signature = signed.signature
        assert isinstance(signature, bytes)
        return signature.hex()
```

Reply:
done